githubteacher-tampa.md

```diff
@@ -0,0 +1,10 @@
+# Tampa, Fl
+
+### Places Visit
+
+
+
+### Places to Eat
+- Dunedin Brewery for :tacos: & :beer:
+- The Place for sandwiches
+- Demitiri's on the Water  
```

Comment:
No, that is a restaurant, but I will add the Sponge Docks as a place to visit. 👍